Task-20/index.js

```diff
@@ -0,0 +1,60 @@
+class Human {
+    name;
+    age;
+    constructor (name, age) {
+        this.name = name;
+        this.age = age;
+    }
+    information () {
```

Comment:
```suggestion
    logInformation () {
```

Почитайте про неймінг